contracts/src/Rolldown.sol

```diff
@@ -378,7 +382,9 @@ contract Rolldown is
     function process_eth_withdrawal(
         Withdrawal calldata withdrawal
     ) private {
-        bool status = payable(address(this)).balance >= withdrawal.amount;
+        bool enought_funds_in_contract = payable(address(this)).balance >= withdrawal.amount;
+        bool is_account = withdrawal.withdrawalRecipient.code.length == 0;
```

Comment:
I presume this fix is just temporary and will only be on the testnet and not on the mainnet... So I do not need to look into how effective this is...

Reply:
it's not. but yes, in a mainnet we will have the updated version without automatic payment, so it's fine for now